<section class="box-content" style="width:100%">
    <h2>Summary</h2>
    <p style="text-align: justify;text-justify: inter-word;">10+ years of research experience, specializing in machine learning, protein function prediction, metagenomic annotation and protein aggregation. Strong publication record with multiple peer-reviewed journal articles and conference proceedings. Passionate about applying computational modeling and ML to understand molecular evolution and interactions shaping interspecies dynamics.
    </p>
    
</section>

<section class="box-content">
    <h2>Timeline</h2>
    <ul class="timeline">
        <li style="">
            <span class="" style="font-weight:bold;">Postdoctoral Fellow</span>
            <a class="float-right">2023-Present</a>
            <p>Emory University, GA, USA </p>
        </li>
        <li style="">
            <span class="" style="font-weight:bold;">Postdoctoral Associate</span>
            <a class="float-right">2022-2023</a>
            <p>Rutgers University, NJ, USA</p>
        </li>
        <li style="">
            <span class="" style="font-weight:bold;">Project Associate, IIT Madras</span>
            <a class="float-right">2022</a>
            <p>Indian Institute of Technology Madras, India</p>
        </li>
        <li style="">
            <span class="" style="font-weight:bold;">M.S. & Ph.D.  Computational Biology</span>
            <a class="float-right">2014-2022</a>
            <p>Indian Institute of Technology Madras, India <br>
            <i>Dissertation Title</i>: Sequence and structural studies of aggregation prone regions in proteins: Development of a prediction method and large-scale analysis</p>
        </li>
        <li style="background-color:rgb(245,245,245)">
            <span class="">Associate</span>
            <a class="float-right">2013-2014</a>
            <p>Cognizant Technology Solutions, India </p>
        </li>
        <li style="background-color:rgb(245,245,245)">
            <span class="">Programmar Analyst</span>
            <a class="float-right">2010-2013</a>
            <p>Cognizant Technology Solutions, India </p>
        </li>
        <li style="">
            <span class="" style="font-weight:bold;">B. Tech. Industrial Biotechnology</span>
            <a class="float-right">2006-2010</a>
            <p>Anna University, India <br>
            <i>Dissertation Title</i>: Cloning and expression of the synthetic insulin gene in Kluyveromyces lactis expression system</p>
        </li>
    </ul>
        
</section>

<!--section class="box-content">
    <h2>Experiences</h2>
    <ul class="timeline">
        <li>
            <div class="experience">
                <h3 class="collapsible">Postdoctoral Fellow, Emory Unveristy, GA, USA 
                    <p class="float-right"><span style="color:white;">2023-present</span></p>
                </h3>
                <div class="collapsible-content">
                    <p>Details about the first publication, including authors, journal, and findings.</p>
                </div>
            </div>
            
        </li>
        <li>
            <div class="experience">
                <h3 class="collapsible">Postdoctoral Associate, Rutgers University, NJ, USA
                    <p class="float-right"><span style="color:white;">2022-2023</span></p>
                </h3>
                <div class="collapsible-content">
                    <p>Details about the second publication, including authors, journal, and findings.</p>
                </div>
            </div>
        </li>
        <li>
            <div class="experience">
                <h3 class="collapsible">Project Associate, IIT Madras
                    <p class="float-right"><span style="color:white;">2021-2022</span></p>
                </h3>
                <div class="collapsible-content">
                    <p>Details about the second publication, including authors, journal, and findings.</p>
                </div>
            </div>
        </li>
        <li>
            <div class="experience">
                <h3 class="collapsible">Software QA & QA Lead
                    <p class="float-right"><span style="color:white;">2010-2014</span></p>
                </h3>
                <div class="collapsible-content">
                    <p>Details about the second publication, including authors, journal, and findings.</p>
                </div>
            </div>
        </li>        
    </ul>
        
</!--section-->


<section class="box-content" style="background-color:rgb(255,250,255);">
    <h2>Tool developments</h2>
    <div class="project" >
        <h3 class="">REBEAN: Read Embedding Based Enzyme ANnotation</h3>
        <div class="project-panel">
            <p>REBEAN is a DNA language Model for enzymatic annotation of sequencing reads. REBEAN is built for discovery of enzymatic function in both known and unknown sequence space mitigating drawbacks of alignment and translation-based approaches.</p>
            <br>
                
            <p><span><strong>Webserver: </strong></span><a href="https://services.bromberglab.org/rebean/submit" target="_blank">https://services.bromberglab.org/rebean/submit</a>
                <br>
                <span><strong>Source (private): </strong></span><a href="https://bitbucket.org/bromberglab/rebeancore/src/main/" target="_blank">https://bitbucket.org/bromberglab/rebeancore/src/main/</a>
                <br>
                <span><strong>Publication: </strong></span><a href="http://dx.doi.org/10.1101/2024.12.10.627786" target="_blank">bioRxiv</a></p>
            </div>
    </div>
    <div class="project" >
        <h3 class="">ANuPP: Aggregation Nucleation Prediction in Peptides and Proteins</h3>
        <div class="project-panel">
            <p>Aggregation Nucleation Prediction in Peptides and Proteins (ANuPP) is a ensemble-classifier developed and trained to identify amyloid-fibril forming peptides and regions in protein sequences.</p>
            <br>
                
            <p><span><strong>Webserver: </strong></span><a href="https://web.iitm.ac.in/bioinfo2/ANuPP/" target="_blank">https://web.iitm.ac.in/bioinfo2/ANuPP/</a>
                <br>
                <span><strong>Source (private): </strong></span><a href="https://github.com/rpkarandev/ANuPP" target="_blank">https://github.com/rpkarandev/ANuPP</a>
                <br>
                <span><strong>Publication: </strong></span><a href="https://doi.org/10.1016/j.jmb.2020.11.006" target="_blank">Prabakaran, R., Rawat, P., Kumar, S., & Michael Gromiha, M. (2021). ANuPP: A Versatile Tool to Predict Aggregation Nucleating Regions in Peptides and Proteins. Journal of molecular biology, 433(11), 166707.</a></p>
            </div>
    </div>
     
</section>

<section class="box-content" style="background-color:rgb(255,255,250);">
    <h2>Database developments</h2>
    <div class="project" >
        <h3 class="">YAbS: The Antibody Society's Antibody Therapeutics Database</h3>
        <div class="project-panel">
            <p>A manually curated resource offering detailed information on 1000+ therapeutic antibodies and their clinical status - developed by The Antibody Society.</p>
            <br>
                
            <p><span><strong>Database: </strong></span><a href="https://db.antibodysociety.org/" target="_blank">https://db.antibodysociety.org/</a>
            <br>
            <span><strong>Publication: </strong></span><a href="https://doi.org/10.1080/19420862.2025.2468845" target="_blank">Rawat, P., Crescioli, S., Prabakaran, R., Sharma, D., Greiff, V., & Reichert, J. M. (2025). YAbS: The Antibody Society's antibody therapeutics database. mAbs, 17(1), 2468845.</a></p>
        </div>
    </div>
    <div class="project" >
        <h3 class="">Ab-Cov: A curated database for binding affinity and neutralization profiles of coronavirus-related antibodies</h3>
        <div class="project-panel">
            <p>Ab-CoV contains manually curated experimental binding affinity (KD) and neutralization profile (IC50 and EC50) of coronavirus-related antibodies along with in silico mutational scanning of epitope and paratope region for known structures and viral protein features.</p>
            <br>
                
            <p><span><strong>Database: </strong></span><a href="https://web.iitm.ac.in/bioinfo2/ab-cov/" target="_blank">https://web.iitm.ac.in/bioinfo2/ab-cov/</a>
                <br>
                <span><strong>Publication: </strong></span><a href="https://doi.org/10.1093/bioinformatics/btac439" target="_blank">Rawat, P., Sharma, D., Prabakaran, R., Ridha, F., Mohkhedkar, M., Janakiraman, V., & Gromiha, M. M. (2022). Ab-CoV: a curated database for binding affinity and neutralization profiles of coronavirus-related antibodies. Bioinformatics (Oxford, England), 38(16), 4051–4052.</a></p>
            </div>
    </div>
    <div class="project" >
        <h3 class="">CPAD 2.0: Curated Protein Aggregation Database</h3>
        <div class="project-panel">
            <p>A curated databses of aggregating peptides, proteins and aggregation prone regions.</p>
            <br>
                
            <p><span><strong>Database: </strong></span><a href="https://web.iitm.ac.in/bioinfo2/cpad2" target="_blank">https://web.iitm.ac.in/bioinfo2/cpad2</a>
                <br>
                <span><strong>Publication: </strong></span><a href="https://doi.org/10.1080/13506129.2020.1715363" target="_blank">Rawat, P., Prabakaran, R., Sakthivel, R., Mary Thangakani, A., Kumar, S., & Gromiha, M. M. (2020). CPAD 2.0: a repository of curated experimental data on aggregating proteins and peptides. Amyloid : the international journal of experimental and clinical investigation : the official journal of the International Society of Amyloidosis, 27(2), 128–133.</a></p>
        </div>
    </div>
</section>


<section class="box-content" style="background-color:rgb(250,255,255);">
    <h2>Teaching experience</h2>
    <ul>
        <li></li>
        <li> 2024-2025 Guest Lecture - Introduction to Bioinformatics, Emory University, GA, USA </li>
        <li> 2023-2025 Mentored three undergraduates and master students on their thesis</li>
        <li> 2019 Crash courses - Programming using Python, Machine-learning and Statistics, Protein Bioinformatics lab, IIT Madras, India </li>
        <li> 2018-2021 Mentored two undergraduates on their Honour thesis</li>
        <li> 2018-2020 Teaching assistant - Bioinformatics: Algorithms and Applications, NPTEL, IIT Madras, India </li>
        <li> 2017 Workshop - "Data analysis and application of machine-learning for ligand binding affinity
            prediction using Python", BioFest 2017, IIT Madras, India</li>
        <li> 2015-2019 Teaching assistant - Computational Biology, IIT Madras, India </li>
        <li> 2015-2019 Teaching assistant - Bioinformatics, IIT Madras, India </li>

    </ul>
</section>

<section class="box-content" style="background-color:rgb(240,255,255);">
    <h2>Other mentions</h2>
    <ul>
        <li> 2025 Travel grant - FMS4BIO25 workshop, AAAI Conference on Artificial Intelligence </li>
        <li> 2024 Travel grant - CSHL Biological Data Science meeting at NY, USA </li>
        <li> 2018-2022 Administrator for computing and web servers, Protein Bioinformatics Lab, IIT Madras, India </li>
        <li> 2014-2020 Awards - HTRA fellowship from Ministry of Human Resource Development (MHRD), Government of India
            during MS and PhD </li>
        <li> 2014 GATE Biotechnology (National level Aptitude test, India) - Rank 34 </li>
        <li> 2010 GATE Biotechnology (National level Aptitude test, India) - Rank 14 </li>

    </ul>
</section>